datasets-numenta/src/main/java/org/knowm/datasets/numenta/NumentaDAO.java

```diff
@@ -0,0 +1,79 @@
+package org.knowm.datasets.numenta;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.knowm.datasets.common.business.DatasetsDAO;
+import org.knowm.yank.Yank;
+
+public class NumentaDAO extends DatasetsDAO {
+
+  public static void init(String dataFilesDir) {
+
+    // / Needs to point to Google drive
```

Comment:
you can delete this init method, right? I can add it when/if I actually upload the files.